src/main/java/org/zeromq/SocketType.java

```diff
@@ -33,7 +33,7 @@
      * and do not implement functionality such as auto-reconnection.
      * PAIR sockets are considered experimental and may have other missing or broken aspects.</strong>
      */
-    PAIR(ZMQ.ZMQ_PAIR),
```

Comment:
There is still a goal to be consistent with jzmq. The only common API between them if org.zeromq, it would make things simpler.

Reply:
I would even propose to use zmq.socket.Sockets but I sincerely hope we will find a way to create a common API with jzmq. Then it would be harder if there are too many references to zmq packages.